mesonbuild/utils/universal.py

```diff
@@ -1050,6 +1050,8 @@ def default_prefix() -> str:
         return 'c:/'
     if is_haiku():
         return '/boot/system/non-packaged'
+    if 'HOMEBREW_PREFIX' in os.environ:
+	return os.environ['HOMEBREW_PREFIX']
```

Comment:
I must be being blind - whats the syntax error here?

Reply:
Github here shows the `return` line at the same indentation level as the `if` statement before it. I think you are mixing spaces and tabs indentation. Meson source code uses spaces only.

Reply:
What would an f-string have to do with a syntax error?

> I think you are mixing spaces and tabs indentation. Meson source code uses spaces only.

More importantly, the python programming language itself errors out with:
```
TabError: inconsistent use of tabs and spaces in indentation
```

And the python community overwhelmingly uses spaces only.

(That being said, github shows nothing of the sort to me. It may be a profile setting on how to view tabs defaulting to interpreting it as tantamount to 8 spaces? Weird.)